tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -969,11 +969,8 @@ def test_randint_float1(self):
     def test_randint_float2(self):
         self.generate(6.7, size=(2, 3))
 
-    @pytest.mark.xfail(numpy.__version__ < "2",
-                       reason='XXX: np 2.0: comparisons with OOB '
-                              'ints are broken in numpy < 2'
-                       )
     def test_randint_int64_1(self):
+        # FIXME: May have subtle bug in function signature determination
```

Comment:
There's still a bug in the comparison logic, but at least the bug in randint has been resolved. Could you please remove the comments here?

Reply:
Removed in 2086313283cc272e58871c49a71dbaf4e928971e .